uri.go

```diff
@@ -47,6 +47,7 @@ type URI struct {
 
 	queryArgs       Args
 	parsedQueryArgs bool
+	useOriginalPath bool
```

Comment:
Please add support for this field to `URI.CopyTo()` and`URI.Reset()`.

Reply:
Added